core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -156,6 +156,8 @@
 public class TableOperationsImpl extends TableOperationsHelper {
 
   public static final String CLONE_EXCLUDE_PREFIX = "!";
+  public static final String compCanceledMsg = "Compaction canceled";
+  public static final String tableDeletedMsg = "Table is being deleted";
```

Comment:
I think in most other places in the code, static final variables follow an UPPER_CASE convention.

Reply:
addressed in 5e9767e